src/main/java/uk/gov/pay/adminusers/resources/ServiceResource.java

```diff
@@ -421,6 +421,12 @@ public Response createStripeAgreement(@Parameter(example = "7d19aff33f8948deb97e
                                           @NotNull @Valid StripeAgreementRequest stripeAgreementRequest) throws UnknownHostException {
         LOGGER.info("Create stripe agreement POST request - [ {} ]", stripeAgreementRequest.toString());
 
+        Optional<StripeAgreement> maybeStripeAgreement = stripeAgreementService.findStripeAgreementByServiceId(serviceExternalId);
+        if (maybeStripeAgreement.isPresent()) {
```

Comment:
Although rare, what if a different user resumes the journey? We won't be storing details of the current user that continues the go-live request in this case.

Reply:
Someone has agreed to the terms - and they have essentially completed the go-live at this point. The only reason an agreement acceptance will have been stored and they're able to accept again is if there's been an error submitting the page.